src/client/Microsoft.Identity.Client/AppConfig/AuthorityType.cs

```diff
@@ -4,6 +4,7 @@
 namespace Microsoft.Identity.Client
 {
     /// <summary>
+    /// Enumeration for the AuthorityTypes
     /// </summary>
     internal enum AuthorityType
```

Comment:
There are still many more usages of AuthorityType in MSAL. Can we eliminate them all, i.e. make `public AuthorityType AuthorityType { get; }` a private property of AuthorityInfo?

Reply:
Checked. That will require lot of refactoring. It has been public for a while. So, refactor can be another work item instead of being part of dSTS.